tasks/install.yml

```diff
@@ -25,22 +25,35 @@
 - include: install_glibc_alpine.yml
   when: ansible_os_family == 'Alpine'
 
+- name: Check for SDKMAN installation
+  stat:
+    path: '{{ sdkman_dir }}/bin/sdkman-init.sh'
+  register: sdkman_init
+
 - name: Download SDKMAN
   get_url:
     url: https://get.sdkman.io
-    dest: /tmp/sdkman_script
+    dest: '{{ sdkman_tmp_dir }}/sdkman_script'
```

Comment:
Good suggestion! I'd rather introduce `tempfile` in a separate change, though. That module was introduced in 2.3, so keeping backward compatibility with the previous minor version of Ansible would require some additional work i.e. detecting the Ansible version at runtime and running certain tasks depending on the detected major/minor version. Like so...

```yaml
- name: Create tmp file
  tempfile: ...
  when: ansible_version.major > 2 or (ansible_version.major == 2 and ansible_version.minor > 3)
```

All for it, but I'd rather introduce it later.